package.json

```diff
@@ -38,7 +37,8 @@
     "ember-disable-prototype-extensions": "^1.0.0"
   },
   "dependencies": {
-    "ember-cli-babel": "^4.0.0"
+    "ember-cli-babel": "^4.0.0",
+    "ember-cli-htmlbars": "^0.7.6"
```

Comment:
FYI, this needs to be moved to regular dependencies otherwise cli would just barf at me when I tried to start the app. The debug message told me to move it here.